### PR TITLE
DDCE-267: updated the logs

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: app/controllers/ExclusionListController.scala [hunk 1]
```diff
@@ -93,10 +93,10 @@ class ExclusionListController @Inject()(
       if (exclusionsAllowed) {
         val iabdTypeValue = uriInformation.iabdValueURLDeMapper(iabdType)
         val staticDataRequest = for {
-          year                                              <- validateRequest(isCurrentTaxYear, iabdType)
-          currentYearList: (Map[String, String], List[Bik]) <- bikListService.currentYearList
-          nextYearList: (Map[String, String], List[Bik])    <- bikListService.nextYearList
-          currentYearEIL: List[EiLPerson]                   <- eiLListService.currentYearEiL(iabdTypeValue, year)
+          year <- validateRequest(isCurrentTaxYear, iabdType)
+          //currentYearList: (Map[String, String], List[Bik]) <- bikListService.currentYearList
```
Comment (Ufuoma-Daniel) [inline app/controllers/ExclusionListController.scala:97]: Commented out code.

File: app/controllers/ExclusionListController.scala [hunk 2]
```diff
@@ -264,7 +264,7 @@ class ExclusionListController @Inject()(
     individualSelectionOption: Option[String])(implicit request: AuthenticatedRequest[_]): Result =
     listOfMatches.size match {
       case 0 =>
-        Logger.error("Matches are zero size")
+        Logger.error("[ExclusionListController][searchResultsHandleValidResult] List of mathces is empty")
```
Comment (Ufuoma-Daniel) [inline app/controllers/ExclusionListController.scala:267]: We see this error a bit in prod, is this something you'd view as an error level log or is it expected behaviour? Also, "matches" > "matches"

File: app/controllers/ExclusionListController.scala [hunk 3]
```diff
@@ -454,7 +454,9 @@ class ExclusionListController @Inject()(
       RegistrationList(None, List(RegistrationItem(iabdType, active = false, enabled = false)))
 
     Logger.info(
-      s"Committing Exclusion for scheme ${request.empRef.toString} , with employees Optimisitic Lock: ${excludedIndividual.get.perOptLock}")
+      s"[ExclusionListController][searchResultsHandleValidResult] Committing Exclusion for scheme ${request.empRef.toString}" +
+        s", with employees Optimisitic Lock: ${excludedIndividual.get.perOptLock}"
```
Comment (Ufuoma-Daniel) [inline app/controllers/ExclusionListController.scala:458]: .get on an option. Any way to get around this?

File: app/controllers/HelpAndContactController.scala [hunk 4]
```diff
@@ -88,15 +88,16 @@ class HelpAndContactController @Inject()(
             case 200 => Redirect(successRedirect).withSession(request.session + (TICKET_ID -> resp.body))
             case 400 => BadRequest(failedValidationResponseContent(Html(resp.body)))
             case 500 => {
-              Logger.warn("submit contact form internal error 500, " + resp.body)
+              Logger.error(
+                s"[HelpAndContactController][submitContactHmrcForm] Submit contact form internal error 500, ${resp.body}")
               InternalServerError(Html(resp.body))
             }
             case status => throw new Exception(s"Unexpected status code from contact HMRC form: $status")
```
Comment (Ufuoma-Daniel) [inline app/controllers/HelpAndContactController.scala:95]: I'd like an error log for the unexpected status. We can at least see what was in the body?

File: app/utils/SystemPropertiesHelper.scala [hunk 5]
```diff
@@ -62,13 +62,16 @@ trait SystemPropertiesHelper {
     }
 
   def doesntExist[T](key: String, defaultvalue: T): T = {
-    Logger.info(s"No system property $key defined. Using default value: $defaultvalue")
+    Logger.error(
```
Comment (Ufuoma-Daniel) [inline app/utils/SystemPropertiesHelper.scala:65]: How frequently do we get this log? Same question for the doesntParse method. I'd be wary if this ends up flooding our kibana logs.

File: app/controllers/ExclusionListController.scala (hunk 1 above)
Comment (fraser-sprott) [reply app/controllers/ExclusionListController.scala:97]: forgot to delete the line it's unused, just commented out to run the tests to verify

File: app/controllers/ExclusionListController.scala (hunk 2 above)
Comment (fraser-sprott) [reply app/controllers/ExclusionListController.scala:267]: I couldn’t honestly say if this is expected behaviour or not

File: app/controllers/ExclusionListController.scala (hunk 3 above)
Comment (fraser-sprott) [reply app/controllers/ExclusionListController.scala:458]: didn't notice the .get, will check and update if required

File: app/utils/SystemPropertiesHelper.scala (hunk 5 above)
Comment (fraser-sprott) [reply app/utils/SystemPropertiesHelper.scala:65]: it seems to come in waves more than anything, 2-3 spikes within the last 14 days

**Pull request merged**